src/scenes/ProgressReports/EditForm/Steps/PreviousStepButton.tsx

```diff
@@ -0,0 +1,33 @@
+import { ArrowBack } from '@mui/icons-material';
+import { Button } from '@mui/material';
+import { StyleProps } from '~/common';
+import { useProgressReportContext } from '../ProgressReportContext';
+
+export const PreviousStepButton = ({ sx }: StyleProps) => {
```

Comment:
This is a copy paste of the next button. Both are fairly worthless now since they are only used in the `StepContainer` component. I'd delete both of these and render there.

You could still assign them to variables to reference twice.

```tsx 
const previousButton = !isFirst && (
  <Button
    variant="outlined"
    color="secondary"
    onClick={previousStep}
    startIcon={<ArrowBack />}
  >
    Previous
  </Button>
);

return (
  <Box ...>{previousButton}{nextButton}</Box>
  ...
  <Box ...>{previousButton}{nextButton}</Box>
);
```